src/main/java/io/xpring/xrpl/model/XRPPayment.java

```diff
@@ -38,35 +38,29 @@
    *
    * @return An {@link Integer} containing the tag that identifies the reason for the payment.
    */
-  @Nullable
-  Integer destinationTag();
+  Optional<Integer> destinationTag();
 
   /**
    * (Optional) Minimum amount of destination currency this transaction should deliver.
    *
    * @return An {@link XRPCurrencyAmount} representing the minimum amount of destination currency this
    *          transaction should deliver.
    */
-  @Nullable
-  XRPCurrencyAmount deliverMin();
+  Optional<XRPCurrencyAmount> deliverMin();
 
   /**
    * (Optional) Arbitrary 256-bit hash representing a specific reason or identifier for this payment.
    *
    * @return A byte array containing a 256-bit hash representing a specific reason or identifier for this payment.
    */
-  @Nullable
   byte[] invoiceID();
 
   /**
-   * Array of payment paths to be used for this transaction.
-   * <p>
+   * (Optional) Array of payment paths to be used for this transaction.
    * Must be omitted for XRP-to-XRP transactions.
-   * </p>
    *
    * @return A {@link List} of {@link XRPPath}s containing the paths to be used for this transaction.
    */
-  @Nullable
   List<XRPPath> paths();
```

Comment:
also probably want to default to an empty `List` here

Reply:
added default behavior to any optional sequence type